### PR TITLE
docs: Update direct routing policy limitations

### PR DESCRIPTION
**Pull request by joestringer**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9015)

### CODE REVIEW

**Comment by joestringer** [comment]
test-me-please

**Comment by joestringer** [comment]
test-docs-please

**Comment by joestringer** [comment]
test-docs-please

File: Documentation/concepts/networking.rst [hunk 1]
```diff
@@ -102,10 +102,9 @@ This is typically achieved using two methods:
   combination with the ``--allocate-node-cidrs`` option then this is configured
   automatically for IPv4 prefixes.
 
-.. note:: Use of direct routing mode currently only offers identity based
-          security policy enforcement for IPv6 where the security identity is
-          stored in the flowlabel. IPv4 is currently not supported and thus
-          security must be enforced using CIDR policy rules.
+.. note:: Use of direct routing mode with advanced policy use cases such as
+          L7 policies is currently beta. Iplease provide feedback and file a
```
Comment (ianvernon) [inline Documentation/concepts/networking.rst]: Iplease --> Please

**Review by ianvernon — CHANGES_REQUESTED** [review]

**Comment by joestringer** [comment]
test-docs-please

**Review by tgraf — APPROVED** [review]

**Comment by brb** [comment]
@joestringer Setting `need-backport/1.5`, as this would reduce some confusion for v1.5 users: https://cilium.slack.com/archives/C53TG4J4R/p1566477425192100?thread_ts=1566438387.191500&cid=C53TG4J4R (via #9035).

**Pull request merged**